From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: bash-aliases-repeat

# DP: Fix bug in Bash_aliases example.

# DP: Fix bug in Bash_aliases example.
---
 examples/startup-files/Bash_aliases | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/examples/startup-files/Bash_aliases b/examples/startup-files/Bash_aliases
index 2abb93e..4bd8e66 100644
--- a/examples/startup-files/Bash_aliases
+++ b/examples/startup-files/Bash_aliases
@@ -41,20 +41,20 @@ repeat ()
 { 
     local count="$1" i;
     shift;
-    for i in $(seq 1 "$count");
+    for i in $(_seq 1 "$count");
     do
         eval "$@";
     done
 }
 
 # Subfunction needed by `repeat'.
-seq ()
+_seq ()
 { 
     local lower upper output;
     lower=$1 upper=$2;
 
     if [ $lower -ge $upper ]; then return; fi
-    while [ $lower -le $upper ];
+    while [ $lower -lt $upper ];
     do
 	echo -n "$lower "
         lower=$(($lower + 1))
